### DISCUSSION TITLE
play kingdom music randomly button

### DISCUSSION THREAD

**Discussion by riemsesy** [sircharlo/meeting-media-manager#277] [Ideas] [answered]
When clicking the music button to start playing Kingdom Music you have to click twice to confirm. 

![image](https://user-images.githubusercontent.com/39587825/166570860-176cf5ee-1103-41d2-8fed-b7f2a5843c5a.png)
Clicking somewhere else should stop that procedure and make the button blue again
![image](https://user-images.githubusercontent.com/39587825/166571427-41679436-f14e-4f9d-a784-c34b9c6682b1.png)

waiting does that too, but it's less intuitive

low/no priority ;-)

**Comment by sircharlo** [comment, marked as answer]
This would actually be surprisingly hard to implement; clicks are notoriously difficult to manage, especially if just trying to listen for when the user clicks "elsewhere".

I'll tag this as wontfix, as I don't think the effort would justify the result.

**Reply by sircharlo** [reply]
I stand corrected. I looked into the popover component of Bootstrap; turns out the hard work is already done for us. 
Committed in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/4b7fc66079b45c0936869fbe04d4c82cfc61c898

**Reply by riemsesy** [reply]
😀👍🏻
